e2e-identity/tests/utils/docker/ldap.rs

```diff
@@ -55,47 +61,42 @@ impl LdapImage {
     }
 }
 
-#[derive(Debug, Default, Clone)]
-pub struct LdapArgs;
-
-impl ImageArgs for LdapArgs {
-    fn into_iterator(self) -> Box<dyn Iterator<Item = String>> {
-        Box::new(
-            vec![
-                "--copy-service".to_string(),
-                "--loglevel".to_string(),
-                "debug".to_string(),
-            ]
-            .into_iter(),
-        )
-    }
-}
-
 impl Image for LdapImage {
-    type Args = LdapArgs;
-
-    fn name(&self) -> String {
-        Self::NAME.to_string()
+    fn name(&self) -> &str {
+        Self::NAME
     }
 
-    fn tag(&self) -> String {
-        std::env::var("OPENLDAP_VERSION").unwrap_or_else(|_| Self::TAG.to_string())
+impl ImageArgs for LdapArgs {
+    fn into_iterator(self) -> Box<dyn Iterator<Item = String>> {
+    fn tag(&self) -> &str {
+        &self.tag
```

Comment:
This is a very strange hunk. :-)

Reply:
Let me factor out these changes into more commits, this is too big for a single one